### PR TITLE
feat: add tags instead of callouts

### LINKED ISSUE #1707: [Docs]: Update the callout for business edition
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Documentation Link

_No response_

### Discord/slack/intercom Link

_No response_

### Describe the problem and improvement.

- Update the callouts as per current plans available on the website.
- Update the terminology to use plan(s).

### PR DESCRIPTION
**Pull request by jnikhila**
## Changes
* A component for tags
* Styles for different types of tags
* Updates to pages to remove callouts and add tags.

## Fixes
#1707 

## Checklist
I have:
- [ ] run the content through Grammarly
- [ ] linked to sample apps when relevant
- [ ] added the meta description for each page in the PR
- [ ] minimized the callouts and added only when necessary
- [ ] added the `queryString` parameter to the Tabs (if used)
- [ ] masked PII in images. For example, login credentials, account details, and more
- [ ] added images only when necessary
- [ ] deleted the images that are no longer used for the updated pages in the PR
- [ ] followed the image file naming convention while renaming or adding new images. (Use lowercase letters, dashes between words, and be as descriptive as possible)
- [ ] used the `<figure/>` tag instead of a markdown representation for images
- [ ] added the `<figcaption/>` tag to add a caption to the image
- [ ] added the `alt` attribute in the `` tag
- [ ] verified and updated the cross-references or created redirect rules for the changed or removed content 
- [ ] reviewed and applied the style changes for UI formatting. For example, Bold the UI elements(Buttons on screen) used in the doc.

### CODE REVIEW

**Review by pranayagarwal96 — APPROVED** [review]

File: website/sidebars.js [hunk 1]
```diff
@@ -187,7 +187,7 @@ const sidebars = {
                   ]
```
Comment (danciaclara) [inline website/sidebars.js:187]: why are there changes in this page?

File: website/docs/connect-data/how-to-guides/setup-datasource-environments.md [hunk 2]
```diff
@@ -1,9 +1,20 @@
- # Setup Multiple Datasource Environments
-
-:::info
-The Datasource environments feature is only available on [Business Edition](https://www.appsmith.com/pricing). This feature is still in Beta. If you have questions and would like to test this feature and share feedback, please contact the support team using the chat widget at the bottom right of this page. 
-:::
-
+---
+title: Setup Multiple Datasource Environments
```
Comment (danciaclara) [inline website/docs/connect-data/how-to-guides/setup-datasource-environments.md:2]: Screenshot 2023-09-21 at 21 50 04

looks odd on this page

Comment (danciaclara) [reply website/docs/connect-data/how-to-guides/setup-datasource-environments.md:2]: also this is out of beta right?

**Review by danciaclara — CHANGES_REQUESTED** [review]

File: website/docs/connect-data/how-to-guides/setup-datasource-environments.md (hunk 2 above)
Comment (jnikhila) [reply website/docs/connect-data/how-to-guides/setup-datasource-environments.md:2]: Not sure. Pranay Agrawal will confirm. For now, I have replicated the call-outs into tags.

File: website/docs/getting-started/setup/installation-guides/kubernetes/configure-high-availability.mdx [hunk 3]
```diff
@@ -1,16 +1,26 @@
...
 description: Follow these steps to configure high availability for Appsmith on Kubernetes.
+title:  Configure High Availability and Scalability
+hide_title: true
 ---
+
+<div className="tag-wrapper">
+ <h1> Configure High Availability and Scalability</h1>
+
+<Tags
+tags={[
```
[the remark this answers is not in the document]
Comment (jnikhila) [reply website/docs/getting-started/setup/installation-guides/kubernetes/configure-high-availability.mdx:11]: The long titles on smaller screens will be visible like this. I will look into css.

File: website/sidebars.js (hunk 1 above)
Comment (jnikhila) [reply website/sidebars.js:187]: Which page?

Comment (danciaclara) [reply website/sidebars.js:187]: sidebar

Comment (jnikhila) [reply website/sidebars.js:187]: looks like formatting changes. It's the same as the main version. I cross-verified.

File: website/docs/getting-started/setup/installation-guides/kubernetes/configure-high-availability.mdx (hunk 3 above)
[the remark this answers is not in the document]
Comment (jnikhila) [reply website/docs/getting-started/setup/installation-guides/kubernetes/configure-high-availability.mdx:11]: fixed.

File: website/docs/connect-data/how-to-guides/setup-datasource-environments.md (hunk 2 above)
Comment (jnikhila) [reply website/docs/connect-data/how-to-guides/setup-datasource-environments.md:2]: We can remove the beta tag once confirmed - @pranayagarwal96 - Please confirm this.

**Review by vasanthappsmith — DISMISSED** [review]

**Review by pranayagarwal96 — APPROVED** [review]

**Review by danciaclara — APPROVED** [review]

**Pull request merged**